bin/cheribsdtest/cheribsdtest.h

```diff
@@ -77,6 +77,8 @@ struct cheribsdtest_child_state {
 	int		ccs_signum;
 	int		ccs_si_code;
 	int		ccs_si_trapno;
+	void		*ccs_si_addr;
+	void		*ccs_si_addr_expected;
```

Comment:
Extremely minor nit: maybe move this above the comment "filled in by the child signal handler" since it's more likely to be set by the test case itself?

Reply:
Good point, I missed that.

Reply:
I've actually moved it under 'Fields filled in by the test itself'.